pkgs/top-level/go-packages.nix

```diff
@@ -45,13 +45,14 @@ let self = _self // overrides; _self = with self; {
   };
 
   protobuf = buildGoPackage rec {
-    rev = "36be16571e14";
-    name = "goprotobuf-${rev}";
-    goPackagePath = "code.google.com/p/goprotobuf";
-    src = fetchhg {
+    rev = "5677a0e3d5e89854c9974e1256839ee23f8233ca";
+    name = "protobuf-${rev}";
```

Comment:
Can we please keep goprotobuf? Because the protobuf name is already used by the protobuf package.